src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -57,6 +57,16 @@ public static bool CanRemoveParentheses(this ParenthesizedExpressionSyntax node,
                 return true;
             }
 
+            if (expression is StackAllocArrayCreationExpressionSyntax)
```

Comment:
Also check for ImplicitStackAllocArrayCreation. they both have the same behviour here.

Reply:
Thanks for review, I'll update that.